SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -42,7 +42,7 @@ void SpatialLoadBalanceEnforcer::AuthorityChanged(const Worker_AuthorityChangeOp
 			// TODO(zoning): There are still some entities being created without an authority intent component.
 			// For example, the Unreal created worker entities don't have one. Even though those won't be able to
 			// transition, we should have the intent component on them for completeness.
-		 	UE_LOG(LogSpatialLoadBalanceEnforcer, Log, TEXT("Requested authority change for entity without AuthorityIntent component. EntityId: %lld"), AuthOp.entity_id);
+		 	UE_LOG(LogSpatialLoadBalanceEnforcer, Log, TEXT("(%s) Requested authority change for entity without AuthorityIntent component. EntityId: %lld"), *WorkerId, AuthOp.entity_id);
```

Comment:
Nit - I think we prefer to put dynamic parts of the message at the end rather than at the beginning. We are very inconsistent about this but try to do it for new stuff.

Reply:
That's actually the opposite of most of the debugging in the load balancing. I was actually removing all of those, but because Squid doesn't have logging (and so the SpatialLogOutput is empty) I was adding it back in for debugging on Squid.

Reply:
I think there were quite a few messages that only really make sense to put things at the end. Generally when you want a load of metadata like, entity ID, actor name, request ID, and status message all in one log.

It got briefly discussed a while ago as something to adopt although I wouldn't be surprised if not enforced. The rational was being able to copy and find a message without worrying about what might be dynamic (although it's often obvious).
It might be worth deciding on this offline though if we've been inconsistent recently.

Reply:
Happy to discuss offline. In this case, the ID is being used as a way to filter multiple messages from different workers, so putting it at the beginning mirrors the SpatialLogOutput.